prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+
+const firstElement = 'html';
+console.log('the first element of the array is:', firstElement);
```

Comment:
Bracket notation should be used to log the correct element eg. to print the first element it would be `languages[0]`

Reply:
Got it. Here is a image of the updated code. I am not sure why I can not create another pull request?
How can I update the code here on github?
Thank you :)

Reply:
<img width="640" alt="Screenshot 2024-10-07 at 3 14 46 PM" src="https://github.com/user-attachments/assets/829c9851-d9cb-4fac-8c8a-3a8da7ab9633">